build_debian.sh

```diff
@@ -188,7 +188,20 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     usbutils                \
     pciutils                \
     iptables-persistent     \
-    logrotate
+    logrotate               \
+    grub2-common
```

Comment:
It already is
Reason that I mentioned it explicitly is that apt-get install -f removes sonic-utilities instead of installing grub2-common